### PR TITLE
Dashboard shortcut widget

### PR DESCRIPTION
**Pull request by sleidig**
~see issue~

### Visible/Frontend Changes
- [x] New dashboard widget: `"DashboardShortcutWidget"` available, showing simple table of shortcuts for easier navigation 

### Architectural/Backend Changes
- [x] some basic refactorings (see commits)

_Merge without squashing to keep the refactoring commits in the git history_

### CODE REVIEW

File: src/app/core/dashboard-shortcut-widget/dashboard-shortcut-widget/dashboard-shortcut-widget.component.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * A simple list of shortcuts displayed as a dashboard widget for easy access to important navigation.
+ */
+@Component({
+  selector: "app-dashboard-shortcut-widget",
+  templateUrl: "./dashboard-shortcut-widget.component.html",
+  styleUrls: ["./dashboard-shortcut-widget.component.scss"],
+})
+export class DashboardShortcutWidgetComponent
+  implements OnInit, OnInitDynamicComponent {
+  /** entries representing each representing one line displayed as a shortcut */
```
Comment (TheSlimvReal) [inline src/app/core/dashboard-shortcut-widget/dashboard-shortcut-widget/dashboard-shortcut-widget.component.ts:15]:
```suggestion
  /** entries, each representing one line displayed as a shortcut */
```

**Review by TheSlimvReal — CHANGES_REQUESTED** [review]
Looks fine, just a small typo. 
However, the styling of other widgets got(?) broken.
widget-shortcut
Also I am wondering who exactly is targeted by this widget? Phone users?

File: src/app/core/dashboard-shortcut-widget/dashboard-shortcut-widget/dashboard-shortcut-widget.component.ts (hunk 1 above)
Comment (sleidig) [reply src/app/core/dashboard-shortcut-widget/dashboard-shortcut-widget/dashboard-shortcut-widget.component.ts:15]: :+1:

**Comment by sleidig** [comment]
Thanks for pointing out the styling issue - somehow I missed the BMI dashboard.

The feature is intended especially for mobile users but also for more specific quick actions on desktop, like directly navigation to the form to create a new child

**Pull request merged**